snap/hooks/configure

```diff
@@ -302,3 +302,9 @@ then
   systemctl restart snap.${SNAP_NAME}.daemon-apiserver
 fi
 
+if [ -f "${SNAP_DATA}/bin/cilium" ]
```

Comment:
Does this need to be `-L`? That's what the disable script uses:

https://github.com/ubuntu/microk8s/blob/b4b721cbdad1018d7e2e09895864e28e71abbd67/microk8s-resources/actions/disable.cilium.sh#L9